StaticAnalyzer/views/android.py

```diff
@@ -1055,7 +1079,186 @@ def Strings(APP_FILE, APP_DIR, TOOLS_DIR):
         return dat
     except:
         PrintException("[ERROR] Extracting Strings from APK")
-
+# Esteve 14.08.2016 - begin - Pirated and Malicious App Detection with APKiD 
+def APKiD(APP_FILE,APP_DIR,TOOLS_DIR,APP_NAME):
+    try:
+        print "[INFO] Detecting Packers, Obfuscators, Compilers, and other stuff with APKiD"
+        RET=''
+        apkid=TOOLS_DIR+'apkid'
+        args=[apkid,'-j',APP_DIR+APP_FILE]
+        dat=ast.literal_eval(subprocess.check_output(args))
+        for key1, value1 in dat.items():
+            if key1.find('!') == -1:
+                file=APP_NAME
+            else:
+                file=key1.split('!')[1]
```

Comment:
We need to do call the escape funtion on untrusted data like `file`
see: https://github.com/ajinabraham/Mobile-Security-Framework-MobSF/blob/master/StaticAnalyzer/views/android.py#L73

I see that you are using `safe` attribute in here: https://github.com/EsteveM/Mobile-Security-Framework-MobSF/blob/80fe2964bfcf68fdb28b0c52a96c9b748b3ce848/templates/static_analysis.html#L723

The data needed to be escaped to prevent a possible XSS.